package.json

```diff
@@ -4,7 +4,9 @@
 		"build": "lerna run build && lerna run build:types",
 		"build:csb": "lerna run --no-bail --ignore 'clayui.com' build && lerna run --no-bail --ignore 'clayui.com' build:types",
 		"compile": "lerna run compile",
-		"ci": "yarn lint && yarn format:check && yarn test",
+		"checkDeps": "node scripts/check-deps/check-dependencies.js",
```

Comment:
I think we might want to add `yarn build` here and remove it from ci. Otherwise we might forget to build if we are checking this locally.

Reply:
Sounds reasonable, although the only bummer is that `build` is super slow (not sure if it gets any faster on 2nd, 3rd etc run either 😢).